src/amo/components/UserProfile/styles.scss

```diff
@@ -63,8 +66,17 @@ $avatar-size: 64px;
 }
 
 .UserProfile-addons-by-author {
-  @include margin(0, 0, 0, 12px);
+  @include margin(12px, 0, 0, 0);
 
-  max-width: 60%;
   width: 100%;
+
+  @include respond-to(large) {
+    @include margin(0, 0, 0, 12px);
+
+    max-width: calc(100% - 300px - 12px);
+  }
+
+  @include respond-to(extraLarge) {
+    max-width: calc(65% - 12px);
```

Comment:
Why do we need a special `max-width` for `extraLarge`?

Reply:
basically when screen size is over 900px (`extraLarge`) the left side is over 300px so to coordinate the right side with that i adjusted max-width to work with the percentage (vs the 100px -300px that is set to work with the `large` bp) .. does that make sense?

Reply:
Kinda sorta. Where did the number `65%` come from?

Reply:
@rebmullin Would be really handy to add a comment with an explanation for what the magic numbers used for things like this, might save some future head-scratching.